Streetcode/Streetcode.BLL/MediatR/Timeline/TimelineItem/GetByStreetcodeId/GetTimelineItemsByStreetcodeIdHandler.cs

```diff
@@ -23,7 +23,9 @@ public async Task<Result<IEnumerable<TimelineItemDTO>>> Handle(GetTimelineItemsB
         var timelineItems = await _repositoryWrapper.TimelineRepository
             .GetAllAsync(
                 predicate: f => f.StreetcodeId == request.StreetcodeId,
-                include: ti => ti.Include(til => til.HistoricalContexts));
+                include: ti => ti
+                    .Include(til => til.HistoricalContextTimelines)
+                        .ThenInclude(x => x.HistoricalContext)!);
```

Comment:
why do you need to get include many-to-many entities too?

Reply:
i need this to get historical context